parts/linux/cloud-init/artifacts/aks-logrotate.sh

```diff
@@ -3,10 +3,10 @@
 # This will be saved on the target VM within /usr/local/bin/logrotate.sh and invoked by logrotate.service
 
 # Clean non existent log file entries from status file
-cd /var/lib/logrotate
+cd /var/lib/logrotate || { echo "Failed to change directory to /var/lib/logrotate. Exiting."; exit 1; }
```

Comment:
this seems silly, could we just `mkdir -p /var/lib/logrotate` and omit this error condition?

Reply:
Sounds good. Done